src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/CallInstruction.cs

```diff
@@ -122,6 +122,9 @@ public static CallInstruction Create(MethodInfo info, ParameterInfo[] parameters
 #endif
         }
 
+        [DynamicDependency("GetValue", typeof(Array))]
```

Comment:
I would just replace the `arrayType.GetMethod` below with `typeof(Array).GetMethod` because that's what it will do anyway because the method doesn't exist on the array itself and needs to go to the base class.

It doesn't need DynamicDependeny+Suppression (which really should only be used in workarounds) and it's even marginally faster:


|          Method |     Mean |   Error |  StdDev |
|---------------- |---------:|--------:|--------:|
|   FromArrayType | 562.5 ns | 2.47 ns | 2.19 ns |
| FromSystemArray | 560.1 ns | 5.00 ns | 4.68 ns |

```csharp
BenchmarkRunner.Run(Assembly.GetExecutingAssembly());

public class Abc
{
    Type TheType = typeof(int[]);

    [Benchmark]
    public MethodInfo FromArrayType() => TheType.GetMethod("GetValue", new[] { typeof(int) });

    [Benchmark]
    public MethodInfo FromSystemArray() => typeof(Array).GetMethod("GetValue", new[] { typeof(int) });
}
```